src/bindgen/ir/function.rs

```diff
@@ -185,6 +191,33 @@ impl Function {
                 reserved::escape(name);
             }
         }
+
+        // Rename pointer arguments which need to use the C-array notation
+        if let Some(tuples) = self.annotations.list("ptrs-as-arrays") {
+            let mut ptrs_as_arrays: HashMap<String, String> = HashMap::new();
+            for str_tuple in tuples {
+                let parts: Vec<&str> = str_tuple[1..str_tuple.len() - 1]
+                    .split(';')
+                    .map(|x| x.trim())
+                    .collect();
+                debug_assert!(parts.len() == 2);
```

Comment:
What guarantees that the user actually writes two parts?

This should probably be a proper `assert_eq!` with an error message, or a warning (but then it should be handled gracefully).